contents/spotlight/startup-langfuse.md

```diff
@@ -0,0 +1,65 @@
+
+> **Putting it into practice:** Pivoting successfully depends on [finding a problem](/founders/product-market-fit-game#level-1---find-a-significant-problem-to-work-on) that is both solvable and _worth_ solving. We know. [We did it six times](founders/story-about-pivots). 
+
+### Take advantage of the Y Combinator community
+
+Among the many perks and benefits which are available to teams in Y Combinator are regular office hours and group sessions with partners and other founders. These are completely voluntary, but invaluable for collecting product feedback and advice. 
+
+“Those sessions were super helpful because we got the advice we needed and learned how to interpret signals from our users,” says Marc. “The whole community was helpful, actually. If there’s one thing I wish I’d done more of then it would be taking more time to connect with other founders in the batch.”
+
+“A lot of the content you get in Y Combinator is just available to everyone on [YouTube](https://www.youtube.com/@ycombinator). But the community and the connections you make with other founders? You can only get that in the batch. It’s important to make the most of it.”
+
+> **Putting it into practice:** In addition to connecting with other founders it’s worth keeping an eye on Bookface for exclusive offers. [Like this one](/yc-onboarding).
+
+### Go where you have the home-turf advantage
+
+Although Y Combinator takes place in San Francisco, Langfuse’s founding team are all from Berlin. They relocated to the USA for the duration of the batch, then returned to Germany once the batch was over.
+
+“We are always debating moving back to San Francisco,” admits Marc. “But at the same time, we’re just very productive here. We know good engineers here and, while we could get by remotely, we want to build the initial team in an office. We go back to San Francisco from time to time to meet users there instead.” 
+
+Moving away from Silicon Valley may sound like an odd move for a tech startup, but Marc believes it’s been for the best and has helped the team gain a home-turf advantage. 
+
+> **Putting it into practice:** Regardless of whether you prefer an in-person or a remote environment, it’s important to [get everyone together at least once or twice a year](/handbook/company/offsites).
+
+### Build open source to reach enterprises
+
+One of the best early decisions the team made was to make Langfuse completely open source where possible. This helped the team not only benefit from external code contributions, but also avoid scaling up sales functions prematurely. 
+
+“Open source is a great way to get more enterprise adoption,” says Marc. “We wouldn’t have expected to reach large Fortune 5000 companies without a dedicated sales team, but we’ve since realized that teams in those companies frequently adopt open source tools because they give them more visibility and control over data.”
```

Comment:
Fortune 500?

Reply:
```suggestion
“Open source is a great way to get more enterprise adoption,” says Marc. “We wouldn’t have expected to reach large Fortune 500 companies without a dedicated sales team, but we’ve since realized that teams in those companies frequently adopt open source tools because they give them more visibility and control over data.”
```